p2p/swarm.go

```diff
@@ -378,7 +381,7 @@ func (s *swarm) processMessage(ime net.IncomingMessageEvent) {
 
 	err := s.onRemoteClientMessage(ime)
 	if err != nil {
-		s.lNode.Errorf("Err reading message from %v, closing connection err=%v", ime.Conn.RemotePublicKey(), err)
+		s.lNode.Error("Err reading message from %v, closing connection err=%v", ime.Conn.RemotePublicKey(), err)
```

Comment:
can we add support for Errorf in the new solution?

Reply:
in the new solution `Error` works like `Errorf`. it was like this in last version. we rarely used `Errorf`